rmgpy/data/thermoTest.py

```diff
@@ -716,6 +716,52 @@ def testAddPolyRingCorrectionThermoDataFromHeuristicUsingAlkaneTricyclic(self):
         self.assertIn('s2_6_6_ane', polycyclicGroupLabels)
         self.assertIn('s2_3_6_ane', polycyclicGroupLabels)
 
+    def testGetBicyclicCorrectionThermoDataFromHeuristic1(self):
+        """
+        Test bicyclic correction estimated properly from heuristic formula
+        """
+        smiles = 'C1=CCC2C1=C2'
+        mol = Molecule().fromSMILES(smiles)
+
+        # extract polyring from the molecule
+        polyring = mol.getDisparateRings()[1][0]
+
+        thermoData = self.database.getBicyclicCorrectionThermoDataFromHeuristic(polyring)
+
+        ringGroups, polycyclicGroups = self.database.getRingGroupsFromComments(thermoData)
+
+        ringGroupLabels = [ringGroup.label for ringGroup in ringGroups]
+        polycyclicGroupLabels = [polycyclicGroup.label for polycyclicGroup in polycyclicGroups]
+
+        self.assertIn('Cyclopentane', ringGroupLabels)
+        self.assertIn('Cyclopropane', ringGroupLabels)
+        self.assertIn('Cyclopentene', ringGroupLabels)
+        self.assertIn('Cyclopropene', ringGroupLabels)
+        self.assertIn('s2_3_5_ane', polycyclicGroupLabels)
+
+    def testGetBicyclicCorrectionThermoDataFromHeuristic2(self):
+        """
+        Test bicyclic correction estimated properly from heuristic formula
```

Comment:
Could you differentiate the docstrings of these two tests? For example, you could add the SMILES of the species you're testing.